test/Microsoft.AspNetCore.WebUtilities.Tests/HttpRequestStreamReaderTest.cs

```diff
@@ -191,6 +195,64 @@ public static void ReadLine_ReadWithNoNewlines()
             Assert.Null(eol);
         }
 
+        [Theory]
+        [MemberData(nameof(HttpRequestNullData))]
+        public static void NullInputsInConstructor_ExpectArgumentNullException(Stream stream, Encoding encoding, ArrayPool<byte> bytePool, ArrayPool<char> charPool)
```

Comment:
`static` means the test class's ctor won't run. Of course, if there isn't one, that's no problem. As long as the test *runs* and it *passes*, it's probably good 😄